app/services/gpo_reminder_sender.rb

```diff
@@ -3,12 +3,19 @@ def send_emails(for_letters_sent_before)
     letter_eligible_range =
       IdentityConfig.store.usps_confirmation_max_days.days.ago..for_letters_sent_before
 
-    profiles_due_for_reminder = Profile.joins(:gpo_confirmation_codes).
-      where(
-        gpo_verification_pending_at: letter_eligible_range,
-        gpo_confirmation_codes: { reminder_sent_at: nil },
-        deactivation_reason: [nil, :in_person_verification_pending],
-      )
+    profiles_due_for_reminder = []
```

Comment:
I suspect that we could make `profiles_due_for_reminder` into a separate method with an implementation of the below. We may just need to pass `letter_eligible_range` in.